rllib/models/tf/layers/noisy_layer.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
```

Comment:
This is not used yet. Will replace the noisy layer with a full keras-style Layer class (getting Rainbow ready for tf2.x eager support).